### PR TITLE
Move import of mozinstall/mozdownload

### PR DESCRIPTION
**Pull request by csnardi**
This caused Travis to error out on all platforms.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/9863)

### CODE REVIEW

**Comment by csnardi** [comment]
Well, this still breaks installing Firefox; everything else is fixed though.

**Comment by w3c-bots** [comment]
# Build BROKEN

Started: 2018-03-06 01:38:39
Finished: 2018-03-06 01:51:33


<h2>Failing Jobs</h2>
<ul>

<li>wptrunner_infrastructure</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25337)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/349585466)

**Comment by csnardi** [comment]
Should be all working now.

File: tools/wpt/commands.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "run": {"path": "run.py", "script": "run", "parser": "create_parser", "help": "Run tests in a browser",
-            "virtualenv": true, "install": ["requests", "mozinstall", "mozdownload"], "requirements": ["../wptrunner/requirements.txt"]},
```
Comment (gsnedders) [inline tools/wpt/commands.json]: @csnardi removing the install here means `./wpt install firefox browser` fails because they aren't installed?

Comment (csnardi) [reply tools/wpt/commands.json]: After the second commit, I don't believe so. I didn't test it locally but Travis failed to install Firefox with only the first commit and then was fine after the second.

Comment (csnardi) [reply tools/wpt/commands.json]: Actually, you're probably right; it still would have failed though prior to this commit I believe.

**Pull request merged**